core/themes/basis/css/component/cards.css

```diff
@@ -13,6 +13,21 @@
   background-color: #f2f2f2;
 }
 
+.update-1-30 .l-top .block-views-promoted-cards-block {
+  margin-top: 0;
+}
+.update-1-30 .l-wrapper > .l-top .block-system-messages + .block-views-promoted-cards-block {
```

Comment:
This causes a slight regression: with the message in place, everything looks good, but after clicking the "dismiss" button, the top padding's missing. A temporary problem, though, as after reloading the page, everything looks good again.

I don't want to block this issue... but it does look weird. Is it possible to remove this @jenlampton ?